test/general/test_patches.py

```diff
@@ -0,0 +1,10 @@
+﻿import unittest
+from worlds.AutoWorld import AutoWorldRegister
+from worlds.Files import AutoPatchRegister
+
+class TestPatches(unittest.TestCase):
+    def test_patch_name_matches_game(self) -> None:
+        for gamename in AutoPatchRegister.patch_types.keys():
+            with self.subTest(game=gamename):
+                self.assertTrue(gamename in AutoWorldRegister.world_types, 
+                                f"Patch \"{gamename}\" does not match the name of any world.")
```

Comment:
Also this is the incorrect assertion.
```suggestion
                self.assertIn(gamename, AutoWorldRegister.world_types,
                                    f"Patch \"{gamename}\" does not match the name of any world.")
```